docs/monitoring/check_data_quality.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Check data quality
+description: A step-by-step monitoring tutorial that shows you how to ensure your data is correctly formatted and unique. Visualize your data using the monitoring dashboard.
+menuWeight: 5.2
+category: guides
+paths:
+    - monitoring/check-data-quality
+---
+
+# [](#check-product-based-data-for-correct-format-and-duplicates) Check product-based data for correct format and duplicates
+
+This example walks you through setting up [monitoring](https://apify.com/apify/monitoring) for an [actor]({{@link actors.md}}) or [task]({{@link actors/tasks.md}}).
+
+## [](#use-case) Use case
+
+You want regularly scrape product data using a single scraper.
+
+You need:
+
+- Data to always be in the [correct format](#validate-data).
+- Alerts if items are [duplicated](#check-for-duplicates).
+- [Notification]({{@link monitoring.md#notifications}}) when your scheduled run finishes successfully.
```

Comment:
Monitoring does not support sending a notification when the run finishes successfully. It only allows sending a notification when the run is TIMED_OUT or FAILED.